espnet2/gan_svs/jets/alignments.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2022 Dan Lim
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from numba import jit
+
+
+class AlignmentModule(nn.Module):
```

Comment:
Our expectation is that you do not copy everything in gan_tts but import them if do not need modification. We may have discussion later if the module needs some levels of changes.